conf/config-default.yaml

```diff
@@ -522,6 +522,14 @@ plugin_attr:
     export_addr:
       ip: 127.0.0.1
       port: 9091
+    # custom_labels:
```

Comment:
Would be better to use:
```
metrics:
    xxx:
       extra_labels:
           label_name_xxx: $label_var
     yyy:
```

So we can add more options to a metric in the future. And we don't need to require the label name should be the variable name.

Reply:
And please add a comment to show that the commented configurations are the example but not the default value.

Reply:
> 

The labels of extra_labels should be a list, because fields of dictionary in lua table are out of order, which would cause the metric full name may change each time it reloads the conf.

Reply:
We can provide order on the implementation side, for example sorting them before using them.